packages/librewolf-app/librewolf-app.pacscript

```diff
@@ -1,37 +1,45 @@
 maintainer="Oren Klopfer <oren@taumoda.com>"
 name="librewolf-app"
 pkgname="librewolf"
-pkgver="119.0.1-1"
+pkgver="120.0.1-1"
 pkgdesc="A fork of Firefox, focused on privacy, security and freedom."
-arch=('amd64' 'arm64')
-if [[ ${CARCH} == "arm64" ]]; then
-  gnuarch="aarch64"
-  hash="746774deae34d915e944df4b063ee093df9ae3cf2ae07434b22c0edec6561fb0"
-else
-  gnuarch="x86_64"
-  hash="ee0b8b77f29cec4c04354e5603b22e1f18ad251f7e7b91afab6d88b1cb2ad2c3"
-fi
-url="https://gitlab.com/api/v4/projects/24386000/packages/generic/librewolf/${pkgver}/LibreWolf.${gnuarch}.AppImage"
+arch=("amd64" "arm64")
+case "${CARCH}" in
+  amd64)
+    gnuarch="x86_64"
+    hash="21e73e405835bd6cfdaecc9930423f68e544eccb52deb7c992e90f5085ca487e"
+    ;;
+  arm64)
+    gnuarch="aarch64"
+    hash="af10277447c2c469d3b6c6a4ea99460de90346edfd8515066a317a954659e0f4"
+    ;;
+  *) return 1 ;;
+esac
```

Comment:
revert this back to the `if` instead of the `case`. `arch=('amd64' 'arm64')` already ensures that there will be no `*)` case, and this is the only place in the script where we are using a return code. I have a feeling this is what is causing the issue.

Reply:
Giving up on `post_remove` for now. This issue needs to be looked into though